### PR TITLE
3/4th safe sprites

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Ports safe and floor safe sprites from https://github.com/tgstation/tgstation/pull/68284
Edits them to include broken variants

## Why It's Good For The Game
Die isometric sprites, die.

Nice pretty sprites are welcome.

## Images of changes
Stolen from the TG PR, but ours are the same just with broken variants that have holes
imageimageimageimage

image
image

## Testing
Sprites

## Changelog
:cl: ReinaCoder S34NW
imageadd: The safe and floor safes have been resprited
/:cl:

### CODE REVIEW

**Comment by Denthamos** [comment]
Will these new sprites have the holes shown from drilling into them?  If no, that presents an issue for visibility so security/command know a safe has been drilled.  If yes, please show such in the PR properly.

**Comment by S34NW** [comment]
> Will these new sprites have the holes shown from drilling into them? If no, that presents an issue for visibility so security/command know a safe has been drilled. If yes, please show such in the PR properly.

Yes, you can see this on the icon diff bot

**Comment by S34NW** [comment]
DNM, it turns out my changes to the DMI got deleted so this PR doesn't actually add the sprites

**Review by Sirryan2002 — APPROVED** [review]
sprite diff looks good

File: paradise.dme [hunk 1]
```diff
@@ -672,8 +672,8 @@
 #include "code\game\machinery\cryopod.dm"
 #include "code\game\machinery\dance_machine.dm"
 #include "code\game\machinery\defib_mount.dm"
-#include "code\game\machinery\dish_drive.dm"
 #include "code\game\machinery\deployable.dm"
+#include "code\game\machinery\dish_drive.dm"
 #include "code\game\machinery\door_control.dm"
```
Comment (hal9000PR) [inline paradise.dme:677]: AAAAAAAAAA

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**